src/pages/404.tsx

```diff
@@ -5,12 +5,8 @@ import { Button } from '@/components/Button'
 
 export default function NotFound() {
   return (
-    <Layout
-      title="Page introuvable"
-      className="px-4 py-16 md:py-24"
-      layoutClassName="bg-white-1"
-    >
-      <div className="text-center max-w-[26rem] m-auto">
+    <Layout title="Page introuvable">
```

Comment:
With this change now the 404 page doesn't have a gray background anymore, is that intentional?

Reply:
my bad